src/js/background.js

```diff
@@ -140,9 +140,25 @@ Badger.prototype = {
 
   // Methods
 
+  loadSeedData: function() {
+    let self = this;
+    utils.xhrRequest(constants.SEED_DATA_LOCAL_URL, function(err, response) {
+      if (!err) {
+        var seed = JSON.parse(response);
+        self.storage.getBadgerStorageObject("action_map").merge(seed.action_map);
+        self.storage.getBadgerStorageObject("snitch_map").merge(seed.snitch_map);
```

Comment:
This is affected by an existing export/import bug (#1972). We lose `snitch_map` entries for all domains that have three entries (i.e. all blocked domains). Since importing an older dataset updates `migrationLevel`, which can cause migrations to be rerun, and at least one existing migration assumes blocked domains to have at least three `snitch_map` entries, importing older datasets can result in all blocked domains in that dataset (and as of this PR, in the pre-trained set as well) to be removed.

Reply:
I'm open both to a quick fix and leaving this alone for now. Just something to consider.